File: Examples/VisualBasic/ApplyLicense/ApplyLicenseByPath.vb
Imports Aspose.CAD
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace ApplyLicense
    Public Class ApplyLicenseByPath
        Public Shared Sub Run()
            ' ExStart:ApplyLicenseByPath
            ' Set path of the license file, i.e. c:\temp\
            Dim dataDir As String = "c:\temp\"

            Dim license As New License()
            license.SetLicense(dataDir & Convert.ToString("Aspose.CAD.lic"))
            ' ExEnd:ApplyLicenseByPath
        End Sub
    End Class
End Namespace

File: Examples/Src/ApplyLicense/ApplyLicenseByPath.cs
using Aspose.CAD;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.CAD.Examples.CSharp.ApplyLicense
{
    public class ApplyLicenseByPath
    {
        public static void Run() 
        {
            //ExStart:ApplyLicenseByPath
            // Set path of the license file, i.e. c:\temp\
            string dataDir = @"c:\temp\";

            License license = new License();
            license.SetLicense(dataDir + "Aspose.CAD.lic");
            //ExEnd:ApplyLicenseByPath
        }
    }
}
